src/api/entities/Asset/types.ts

```diff
@@ -393,14 +396,66 @@ export interface HeldNfts {
   nfts: Nft[];
 }
 
+export type AccreditedValue = {
+  /**
+   * The count or percentage of Asset holders that are accredited
+   */
+  accredited: BigNumber;
+
+  /**
+   * The count or percentage of Asset holders that are not accredited
+   */
+  nonAccredited: BigNumber;
+};
+
+export type AffiliateValue = {
+  /**
+   * The count or percentage of Asset holders that are affiliates
+   */
+  affiliate: BigNumber;
+
+  /**
+   * The count or percentage of Asset holders that are not affiliates
+   */
+  nonAffiliate: BigNumber;
+};
+
+export type JurisdictionValue = {
+  /**
+   * The country code of the jurisdiction
+   * @note null if the jurisdiction is not specified
+   */
+  countryCode: CountryCode | null;
+
+  /**
+   * The count or percentage of Asset holders with the jurisdiction
+   */
+  count: BigNumber;
+};
+
 /**
- * A Transfer Restriction along with its current value
+ * Asset Stat along with its current value
  */
-export interface TransferRestrictionValues {
-  restriction: TransferRestriction;
+export interface TransferStatValues {
```

Comment:
```suggestion
export interface TransferRestrictionStatValues {
```
I think Restriction helps make it clear its not tracking stuff about transfers